main/src/ca/uwaterloo/flix/api/lsp/provider/completion/Completion.scala

```diff
@@ -198,6 +198,14 @@ sealed trait Completion {
         textEdit = TextEdit(context.range, name),
         documentation = None,
         kind = kind)
+    case Completion.UseEnumCompletion(name) =>
+      CompletionItem(
+        sortText = "",
```

Comment:
sortText should probaby equal name, I guess.

Reply:
fixed, just building again to make sure it still works as intended